### DISCUSSION TITLE
Is ant-design 1.7.8 would work with Vue V3?

### DISCUSSION THREAD

**Discussion by eliran-natan** [ant-design/ant-design#36611] [General]
We are currently running on ant-design-vue 1.7.8 and Vue 2.6.14. 
We wish to upgrade to Vue 3. Should we expect any problems with compatibility?

**Comment by zheeeng** [comment]
[Here](https://github.com/vueComponent/ant-design-vue/discussions) is the right place to open discussions about Ant Design Vue.

Accords to its document you should upgrade your ant-design-vue to 3.x to migrate to Vue 3. https://www.antdv.com/docs/vue/migration-v3

**Reply by eliran-natan** [reply]
@zheeeng, true, but for ant-design-vue 2.x to work, we need to upgrade to Vue 3.
This means that we cannot do a step-by-step upgrade. We have to upgrade both on the same go. 
Please correct me if I'm wrong here. Thanks!

**Reply by afc163** [reply]
Wrong place to discusstion, we know nothing about vue and it's UI library.

**Reply by zheeeng** [reply]
According to the official document, migrating from v1 to v2 and upgrading to v3 should work.

https://www.antdv.com/docs/vue/migration-v2

https://www.antdv.com/docs/vue/migration-v3
